shopping_app/lib/screens/home_screen/local_home_screen.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import 'package:shopping_app/items/category_list_single_item.dart';
+import 'package:shopping_app/items/data_search.dart';
+import 'package:shopping_app/models/dummy_data.dart';
+import '../../items/home_screen_top_component.dart';
```

Comment:
What is the difference between relative and absolute path

Reply:
Path From Root to File with complete address will be ABSOLUTE 
Path related with Current File being accessed ( on in which we are working ) to the file needed is RELATIVE does not contain a complete root and all directories. Does not need to traverse back to the root and then to the file needed